frontend/compute-system/src/components/AppStore/AppCard/components/TextsBox.tsx

```diff
@@ -20,7 +20,11 @@ const TextsBox = (props: TextsBoxProps) => {
       className={classes.textBox}
     >
       <Grid item>
-        <Typography variant="h5">{props.title}</Typography>
+        <Typography variant="h5">
+          {props.title && props.title.length > 24
```

Comment:
Do you need `props.title &&`?

Reply:
Just a double checking. Probably not needed, but in my opinion saver.